src/prefect/client/orchestration.py

```diff
@@ -3054,7 +3054,12 @@ async def read_worker_metadata(self) -> Dict[str, Any]:
         return response.json()
 
     async def increment_concurrency_slots(
-        self, names: List[str], slots: int, mode: str, create_if_missing: Optional[bool]
+        self,
+        names: List[str],
+        slots: int,
+        mode: str,
+        create_if_missing: Optional[bool],
+        active: bool = False,
```

Comment:
this seems like an odd place to toggle active/inactive; unless there's a really concrete use case for it here I'm not a fan